appveyor.yml

```diff
@@ -0,0 +1,48 @@
+build: false
+shallow_clone: false
+skip_branch_with_pr: true
+clone_depth: 1
+
+environment:
+
+  global:
+    NOSE_COVER_PACKAGE: "ipykernel"
+
+  matrix:
+    - python: "C:/Python27-x64"
+    - python: "C:/Python27"
+    - python: "C:/Python34-x64"
+    - python: "C:/Python34"
+    - python: "C:/Python35-x64"
+    - python: "C:/Python35"
+    - python: "C:/Python36-x64"
+    - python: "C:/Python36"
+    - python: "C:/Python36-x64"
```

Comment:
Since AppVeyor tends to be very slow, could we cut this down to just py27 and py36? The I'm comfortable with the Python version coverage we have on other, faster CI.